### PR TITLE
Fixing Tech Debts and Sign Up Bug

### PR DESCRIPTION
**Pull request by ravioli0509**
### Description

This PR fixes tech debts such as, 
1. Erasing Logout functions 
2. Displaying other profiles using, bookOwner uuid 
3. Making username unique for edit user 
4. Fixing Signup since unique function wasn't working.

### CODE REVIEW

File: app/src/main/java/com/team41/boromi/dbs/UserDB.java [hunk 1]
```diff
@@ -102,4 +101,31 @@ public User pushUser(User user) {
...
+      );
+    } catch (Exception e) { // Request failed
+      Log.w(TAG, e.getCause());
+      return null;
+    }
+    if (res.isEmpty()) { // No user found
+      return true;
+    } else {
+      return false;
+    }
```
Comment (blchelle) [inline app/src/main/java/com/team41/boromi/dbs/UserDB.java]: Can be simplified to
'''java
return res.isEmpty()
'''

Though I guess that's just preference

**Review by blchelle — APPROVED** [review]
lgtm

File: app/src/main/java/com/team41/boromi/dbs/UserDB.java (hunk 1 above)
Comment (ravioli0509) [reply app/src/main/java/com/team41/boromi/dbs/UserDB.java]: yea true thanks!

Comment (ravioli0509) [reply app/src/main/java/com/team41/boromi/dbs/UserDB.java]: fixed c3eca5d

**Pull request merged**